CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Support `ArrowJson` schema output format in QGL API and CLI commands
 - New `kamu system compact <dataset>` command that compacts dataslices for the given dataset
+### Changed
+- Case insensetive comparisons of <dataset>s, <account>s and <repo>s
```

Comment:
We normally use angle brackets <> to indicate a placeholder for an object name, of certain object type.